indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +32,4 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+    location_data = IndicoLocationField(_("Location"), description=_("Location"))
```

Comment:
The description is not really used since it's updated dynamically. And since I cannot just omit the description parameter, I just wrote "Location". Any better approaches?